UKHO.PeriodicOutputService/UKHO.PeriodicOutputService.API.FunctionalTests/FunctionalTests/POSEndToEndScenarioUpdatePollingTimeOut.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System.Net;
+using FluentAssertions;
+using NUnit.Framework;
+using UKHO.PeriodicOutputService.API.FunctionalTests.Helpers;
+using static UKHO.PeriodicOutputService.API.FunctionalTests.Helpers.TestConfiguration;
+
+namespace UKHO.PeriodicOutputService.API.FunctionalTests.FunctionalTests
+{
+    [Category("POSEndToEndScenarioUpdatePollingTimeOut")]
+    public class POSEndToEndScenarioUpdatePollingTimeOut
+    {
+        private string fssJwtToken;
+        private static readonly POSWebJobApiConfiguration posWebJob = new TestConfiguration().POSWebJobConfig;
+        private static readonly POSFileDetails posDetails = new TestConfiguration().posFileDetails;
+        private static readonly FSSApiConfiguration FSSAuth = new TestConfiguration().FssConfig;
+
+        [OneTimeSetUp]
+        public async Task Setup()
+        {
+            AuthTokenProvider authTokenProvider = new();
+            fssJwtToken = await authTokenProvider.GetFssToken();
+
+            HttpResponseMessage apiResponse = MockHelper.ConfigureFM(posWebJob.MockApiBaseUrl, posWebJob.FMConfigurationUpdatePollingTimeout);
+            apiResponse.StatusCode.Should().Be((HttpStatusCode)200);
+            await CommonHelper.RunWebJob();
+        }
+
+        [Test]
+        public async Task WhenTheExecutedPosWebJobForUpdateTimesOut_ThenCommitInProgressBatchStatusIsReturned()
+        {
+            HttpResponseMessage apiResponse = await GetBatchDetails.GetBatchDetailsEndpoint(FSSAuth.BaseUrl, posDetails.UpdatePollingTimeoutBatchId);
+            apiResponse.StatusCode.Should().Be((HttpStatusCode)200);
+
+            dynamic batchDetailsResponse = await apiResponse.DeserializeAsyncResponse();
+            string batchStatus = batchDetailsResponse.status;
+            batchStatus.Should().Be("CommitInProgress");
+
+            await FileContentHelper.VerifyPosBatches(fssJwtToken);
+        }
+
+        [OneTimeTearDown]
+        public void GlobalTearDown()
+        {
+            //cleaning up the downloaded files from temp folder
+            FileContentHelper.DeleteTempDirectory(posDetails.TempFolderName);
+
+            //cleaning up the stub home directory
+            HttpResponseMessage apiResponse = MockHelper.Cleanup(posWebJob.MockApiBaseUrl);
+        }
+    }
```

Comment:
this seems to be very close to the FullPollingTimeout so i can't see why we have two classes which essentially is a single class with two tests inside.

Reply:
As discussed with Jason, we have a reason to keep all scenarions as separate files. For each test case scenario, we run the webjob and everytime stub creates new batches according to given scenario.

Category attribute is added to run all test case files sequentially instead of parallel. Otherwise, we get 409-conflict from webjob.